data/human/news.txt

```diff
@@ -1557,6 +1557,28 @@ news "republic soldier"
 
 
 
+news "lawyer"
+	location
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+	name
+		word
+			"Lawyer"
+			"Attorney"
+	message
+		word
+			`"`
+		word
+			"The Dereliction and Salvage Act of 2976 states unmanned vessels gain abandoned status as soon as their owners leave the system."
```

Comment:
I dont know much about content but that dude is just randomly stating laws in the spaceport without even one "did you know that.." or some other form of introduction. Isnt that a bit weird?

Reply:
I added more context to the messages.

Reply:
I always interpreted most of the news items as being a tiny snippet of a larger conversation you were having or overheard. It leaves the imagination to fill in what lead to the line you actually hear in the news panel.